.devcontainer/devcontainer.json

```
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/php-mariadb
+{
+    "name": "GLPI docker devcontainer",
+    "dockerComposeFile": "docker-compose.yml",
+    "service": "php",
```

There is no `php` service

```suggestion
    "service": "app",
```